SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,123 @@
+You set the size of chunks for your world in [launch configuration files](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation).
+### Component
+See [SpatialOS component](#spatialos-component).
+### Configuration files
+The configuration files contain information on how elements of your project must work. There are four configuration files:
+* The [launch configuration file - `*.json`](#launch-configuration-file) contains the information that the “launch a deployment” commands use to use to  run a [deployment](#deployment).
+* The [worker configuration file - `*.worker.json`](#worker-configuration-file) tells SpatialOS how to build, launch, and interact with [workers](#workers).
+* The [project definition file - `spatialos.json`](#project-definition-file) 
+* The [worker packages file - `spatialos_worker_packages.json`(SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/reference/file-formats/spatial-worker-packages)
+
+### Command-line tool (CLI)
+See [Spatial command-line tool (CLI)](#spatial-command-line-tool-cli).
+
+### Deployment
+When you want to try out your game, you need to deploy it. This means launching SpatialOS itself. SpatialOS sets up the [world](#spatialos-world) based on a [snapshot](#snapshot), then starts up the [server-workers](#server-workers) needed to run the world. 
+There are two types of deployment: local and cloud.
+Local deployments allow you to start the SpatialOS [Runtime](#spatialos-runtime) locally to test changes quickly. Find out more about local deployments in the [SpatialOS documentation](https://docs.improbable.io/reference/latest/shared/deploy/deploy-local.
+As their name suggests, cloud deployments run in the cloud on [nodes](#node). They allow you to share your game with other people and run your game at a scale not possible on one local machine. Once a cloud deployment is running, you can connect [game clients](#game-client) to it using the [Launcher](#launcher).
+### Entity
+See [SpatialOS entity](#spatialos-entity).
+### Game client
+Not to be confused with [client-worker](#workers).
+A game client is a binary file;  it’s an executable computer program that runs on a player’s computer. A player uses a game client to play your game.  A client-worker is an object instantiated by this executable program when it runs.
+### Game world
+>Not to be confused with [SpatialOS world](#spatialos-world).
+Everything in your Unreal game that a player can see or interact with. 
+### Inspector
+The Inspector is a web-based tool that you use to explore the internal state of a [SpatialOS world](#spatialos-world). It gives you a real-time view of what’s happening in a [local or cloud deployment](#deployment). Among other things, it displays:
+which [workers](#workers) are connected to the deployment.
```

Comment:
```suggestion
* which [workers](#workers) are connected to the deployment.
```

This should be a bullet point, as should the two below. I can't remember if this will also require an extra line break above to make this work.